Include http4k in the Kotlin for the Serverside (#1856)
JetBrains/kotlin-web-site: pages/docs/reference/server-overview.md

* For a more in-depth introduction to the language, check out the [reference documentation](/docs/reference/index.html) on this site and
[Kotlin Koans](/docs/tutorials/koans.html).
* Micronaut also has a lot of well-detailed [guides](https://guides.micronaut.io/tags/kotlin.html), showing how you can build microservices in Kotlin.

> * http4k has [CLI](http://localhost:8000/installation/) to help you generate fully formed projects, or a [starter](https://start.http4k.org) repo to generate an entire CD pipeline using GitHub, Travis and Heroku with a single bash command.

Reviewer: ```suggestion
* http4k provides the [CLI] to generate fully formed projects, and a [starter](https://start.http4k.org) repo to generate an entire CD pipeline using GitHub, Travis, and Heroku with a single bash command.
```
